Dockerfile

```diff
@@ -22,13 +21,18 @@ RUN apk --update add \
     rm /var/cache/apk/* && \
     ln -s /usr/bin/php7 /usr/bin/php
 
+COPY composer.json /usr/src/app/composer.json
```

Comment:
Right. `COPY` in a `Dockerfile` does not have the same interface or behavior as `cp`.

Here's a snippet from the docs:

> `COPY <src>... <dest>`
>
> Multiple <src> resources may be specified but they must be relative to the source directory that is being built (the context of the build).
>
> Each <src> may contain wildcards and matching will be done using Go’s filepath.Match rules.
>
> The <dest> is an absolute path, or a path relative to WORKDIR, into which the source will be copied inside the destination container.

https://docs.docker.com/engine/reference/builder/#copy